### PR TITLE
Some fixed form the Anagrams experiment.

### LINKED ISSUE #5199: Date and Time should not wrap in the table visualization
This task is automatically imported from the old Task Issue Board and it was originally created by James Dunkerley.
Original issue is [here](https://www.pivotaltracker.com/story/show/184382514).
<hr />

Currently, a date entry gets split to multiple lines. Add the tiny block of CSS to stop this.

### LINKED ISSUE #5576: Investigate issues with large integer values from Anagrams experiment.
When testing out a large example various issues occurred with BigInteger values.

This ticket is to triage the issues and work out what needs to be fixed.

### PR DESCRIPTION
**Pull request by jdunkerley**
### Pull Request Description

- Fixes the display of Date, Time_Of_Day and Date_Time so doesn't wrap.
- Adjust serialization of large integer values for JS and display within table.
- Workaround for issue with using `.lines` in the Table (new bug filed).
- Disabled warning on no specified `separator` on `Concatenate`.

Does not include fix for aggregation on integer values outside of `long` range.

### Checklist

Please include the following checklist in your PR:

- [x] The documentation has been updated if necessary.
- [x] All code conforms to the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides.
- All code has been tested:
  - [x] Unit tests have been written where possible.
  - [ ] If GUI codebase was changed: Enso GUI was tested when built using BOTH
        `./run ide build` and `./run ide watch`.

### CODE REVIEW

**Review by farmaazon — APPROVED** [review]
Reviewed only table.js file, as it's owned by me.

File: test/Table_Tests/src/Common_Table_Operations/Aggregate_Spec.enso [hunk 1]
```diff
@@ -1372,6 +1372,11 @@ spec setup =
             tester = expect_column_names ["Concatenate Text"]
             Problems.test_problem_handling action problems tester
 
+        Test.specify "should not fail if trying concatenate unquoted delimiters with no separator" <|
+            column = Concatenate "Text" separator=""
+            result = table.aggregate [column] on_problems=Report_Error
+            result.column_names . should_equal ["Concatenate Text"]
```
Comment (radeusgd) [inline test/Table_Tests/src/Common_Table_Operations/Aggregate_Spec.enso]: Ideally, I'd do it on small table and verify the result too.
```suggestion
        Test.specify "should not fail if trying concatenate unquoted delimiters with no separator" <|
            column = Concatenate "Text" separator=""
            t = table_builder [["Text", ["A", "BC", "def"]]]
            result = t.aggregate [column] on_problems=Report_Error
            Problems.assume_no_problems result
            result.column_names . should_equal ["Concatenate Text"]
            result.at "Concatenate Text" . to_vector . should_equal ["ABCdef"]
```

File: std-bits/table/src/main/java/org/enso/table/data/table/Column.java [hunk 2]
```diff
@@ -120,12 +122,27 @@ public static Column fromItems(String name, List<Value> items) {
...
+      throw new IllegalArgumentException("Repeat count must be positive.");
+    }
+
+    if (repeat == 1) {
+      return fromItems(name, items);
+    }
+
     var totalSize = items.size() * repeat;
+
+    // ToDo: This a workaround for an issue with polyglot layer. #5590 is related.
```
Comment (radeusgd) [inline std-bits/table/src/main/java/org/enso/table/data/table/Column.java]: So cool that now issue numbers work flawlessly in code preview on GH 😁

Comment (radeusgd) [reply std-bits/table/src/main/java/org/enso/table/data/table/Column.java]: It is unclear to me how to "turn off" the mentioned workaround, if I were to try reproducing this bug.

Maybe we can add a variable here `boolean workaround=true;`?

File: test/Table_Tests/src/In_Memory/Table_Spec.enso [hunk 3]
```diff
@@ -124,6 +124,12 @@ spec =
             (js_converted_dates == table.at "enso_dates").to_vector . should_equal [True, True]
             (enso_date_times == table.at "js_dates").to_vector . should_equal [True, True]
 
+        Test.specify "should work with a split Text value" <|
+            words = "The quick brown fox jumps over the lazy dog" . split " "
+            table = Table.new [["words", words]]
+            table.at "words" . storage_type . should_equal Storage.Text
+            table.at "words" . to_vector . should_equal words
```
Comment (radeusgd) [inline test/Table_Tests/src/In_Memory/Table_Spec.enso]: Can we have a note why this test is added? It is a bit curious.

**Review by radeusgd — APPROVED** [review]
Looks all good, just some small suggestions to add.

**Review by hubertp — APPROVED** [review]

File: std-bits/table/src/main/java/org/enso/table/data/table/Column.java [hunk 4]
```diff
@@ -104,8 +105,10 @@ public Column rename(String name) {
    */
   public static Column fromItems(String name, List<Value> items) {
     InferredBuilder builder = new InferredBuilder(items.size());
-    for (Value item : items) {
-      Object converted = Polyglot_Utils.convertPolyglotValue(item);
+    // ToDo: This a workaround for an issue with polyglot layer. #5590 is related.
+    // to revert replace with: for (Value item : items) {
+    for (Object item : items) {
+      Object converted = item instanceof Value v ? Polyglot_Utils.convertPolyglotValue(v) : item;
```
Comment (JaroslavTulach) [inline std-bits/table/src/main/java/org/enso/table/data/table/Column.java:111]: Yes, related to #5590 - glad to see a workaround.

**Pull request merged**